pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -202,6 +185,18 @@ func (s *Server) watchRootCertAndGenKeyCert(names []string, stop <-chan struct{}
 	}
 }
 
+// updatePluggedinRootCertAndGenKeyCert generates new dns certs and notifies keycertbundle
+func (s *Server) updatePluggedinRootCertAndGenKeyCert() error {
```

Comment:
It is not clear from the name and comments

Reply:
Done